community/modules/compute/schedmd-slurm-gcp-v6-nodeset/outputs.tf

```diff
@@ -32,6 +32,11 @@ output "nodeset" {
     EOD
   }
 
+  precondition {
```

Comment:
nit. consider adding precondition that last numeral in topology is divisible by number of gpus.